rules/sun_checks.xml

```diff
@@ -249,7 +249,7 @@
     <module name="VariableDeclarationUsageDistance"/>
     <module name="ImportOrder">
         <property name="option" value="top"/>
-        <property name="groups" value="java, javax, org, com, , com.runtimecollective, com.brandwatch"/>
+        <property name="groups" value="java, javax, org, com, , com.runtimecollective, com.brandwatch, com.crimsonhexagon"/>
```

Comment:
Do two subsequent commas indicate an empty line? If so I think you ought to have more than just one of them:

```
    <option name="IMPORT_LAYOUT_TABLE">
      <value>
        <package name="" withSubpackages="true" static="true" />
        <emptyLine />
        <package name="java" withSubpackages="true" static="false" />
        <emptyLine />
        <package name="javax" withSubpackages="true" static="false" />
        <emptyLine />
        <package name="org" withSubpackages="true" static="false" />
        <emptyLine />
        <package name="com" withSubpackages="true" static="false" />
        <emptyLine />
        <package name="" withSubpackages="true" static="false" />
        <emptyLine />
        <package name="com.runtimecollective" withSubpackages="true" static="false" />
        <emptyLine />
        <package name="com.brandwatch" withSubpackages="true" static="false" />
        <emptyLine />
        <package name="com.crimsonhexagon" withSubpackages="true" static="false" />
        <emptyLine />
      </value>
    </option>
```

Reply:
no - the `<property name="separated" value="true"/>` tag indicates new lines between the groups, the empty part in the groups `, ,` is to indicate "everything else"

Reply:
ah!